tests/servo_test.py

```diff
@@ -41,7 +41,7 @@ def adjust_handler(self, results: List[EventResult]) -> None:
         return "adjusting!"
 
     @before_event(Events.measure)
-    def do_something_before_measuring(self) -> None:
+    def do_something_before_measuring(self, metrics: List[str] = [], control: servox.Control = None) -> None:
```

Comment:
Should use `Control()` instead of None for the sake of consistency

Reply:
The arg is actually unused, which is why I set it to None (unnecessary mem alloc) - but updated it to match other test methods in the same file, anyway. Also dropped "servox." namespace in favor of using the 'from ... import Control' that's already there.

Reply:
Awesome, thanks. This provides greater typing consistency as well since the argument type technically should not be able to accept a None value because its not specified as optional eg. `Optional[Control]`